db/schema.rb

```diff
@@ -144,9 +136,9 @@
     t.integer "form_id", limit: 4, null: false
     t.text "group_hint_translations", limit: 65535
     t.text "group_name_translations", limit: 65535
-    t.integer "group_rank", limit: 4
```

Comment:
I honestly can't remember and I didn't mean to remove it. Can you put it back in please. This must have happened by accident when I was playing around with the production DB. Thank you for catching!